index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>You Tube Video Page</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+    <header class="header-section">
+        <!--navbar-->
+        <div class="logo left">
+            <i id="menu" class="material-icons">menu</i>
+            <img src="images/youtube.png" alt="logo">
+        </div>
+
+        <div class="search center">
+            <form action="#">
+                <input type="text" placeholder="Search">
+                <button type="button"><i class="material-icons">search</i></button>
+            </form>
+            <i class="material-icons mic">mic</i>
+        </div>
+
+        <div class="icons-right">
+            <i class="material-icons">videocam</i>
+            <i class="material-icons">apps</i>
+            <i class="material-icons">notifications</i>
+            <i class="material-icons display-this">account_circle</i>
+        </div>
+    </header>
```

Comment:
Please make sure that there is a line break after each html code block. This is to improve your code readability